dist/tools/ci/print_toolchain_versions.sh

```diff
@@ -91,4 +91,5 @@ for p in avr; do
 done
 printf "%20s: %s\n" "cppcheck" "$(cppcheck_version)"
 printf "%20s: %s\n" "coccinelle" "$(spatch_version)"
+printf "%20s: %s\n" "git" "$(git --version)"
```

Comment:
I guess we can assume git is always available? If it isn't, would ```"$(git --version || echo "missing/error")"``` yield tousable output?

Reply:
though about that, too first - but if there is no git you can't clone/checkout RIOT and never call this script right?

Reply:
I amended a proper version check as for other tools. Though, I doubt someone would download RIOT as a ZIP **and** would then call a script that is supposed to be used by CI (only).